tests/jenkins/collections-lab.groovy

```diff
@@ -0,0 +1,300 @@
+// -------------- Configuration --------------
+// CloudForms
+def opentlc_creds = 'b93d2da4-c2b7-45b5-bf3b-ee2c08c6368e'
+def opentlc_admin_creds = '73b84287-8feb-478a-b1f2-345fd0a1af47'
+def cf_uri = 'https://portal.opentlc.com'
+def cf_group = 'opentlc-access-cicd'
+// IMAP
+def imap_creds = 'd8762f05-ca66-4364-adf2-bc3ce1dca16c'
+def imap_server = 'imap.gmail.com'
+// Notifications
+def notification_email = 'gpteinfrasev3@redhat.com'
+def rocketchat_hook = '5d28935e-f7ca-4b11-8b8e-d7a7161a013a'
```

Comment:
You can get rid of everything related to Rocketchat

Reply:
Removed from this file. Will remove from all file next week.